server/src/domain/server-info/server-info.service.ts

```diff
@@ -60,4 +66,14 @@ export class ServerInfoService {
 
     return serverStats;
   }
+
+  getSupportedMediaTypes(): ServerMediaTypesResponseDto {
+    return {
+      mimeTypes: [
+        ...Object.values(mimeTypes.video),
+        ...Object.values(mimeTypes.image),
+        ...Object.values(mimeTypes.sidecar),
```

Comment:
We probably don't want sidecar mixed in with the rest, otherwise you'll be uploading xmp files.

I think it might make sense to return video as a list, images as a list, and sidecar as a list. They are also technically a list of file extensions not mime types.

Reply:
I use `Object.values` to get the values instead of putting the records into the response so it would look like this

![image](https://github.com/immich-app/immich/assets/27055614/f5409877-665d-4c11-8f30-19d7ff3d5c9d)